controllers/posts_controller.js

```diff
@@ -69,59 +63,36 @@ module.exports.search = (req, res) => {
     }
 
     if (limit)
-        req.query.limit = parseInt(limit);
+        req.query.limit = Number(limit);
```

Comment:
why assigning values to the `query` object?

Reply:
Yeah should probably just use a separate object